openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -20,18 +20,13 @@
 - control_key: SC-2
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        The customer is responsible for separating user functionality
-        including user interface services) from information system management
-        functionality. A successful control response will indicate how user
-        and management functionality access is separated.
-
-        A complete control response is planned. Engineering progress can be
-        tracked via:
-
-        https://issues.redhat.com/browse/CMP-429
+        Application partitioning is implemented within OpenShift configuration by
+        enforcement of Role Based Access Control (RBAC) objects. RBAC objects support
+        separation of user functionality from admin functionality per project, and
+        supports separation of project admin functionality from OpenShift admin functionality.
```

Comment:
Need link to RBAC documentation, but what about namespaces, just using a container, how master/worker nodes are designed for this, etc.?